.chloggen/890.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: breaking
+component: messaging
+note: Rename `messaging.operation` to `messaging.operation.name`.
```

Comment:
This sounds reasonable.

We could have a `messaging.operation.type` enum as required, and `messaging.operation.name` string as recommended. In span names, we could use `messaging.operation.name` if available, and otherwise `messaging.operation.type`. This could avoid wordy customizations around span name like [this](https://github.com/open-telemetry/semantic-conventions/pull/634/files#diff-b4525d57b2e14f26eee734f14cac1c86a3d3a5f8e0b0fdca24c75c585402661fR31).

Let's discuss in the next meeting before updating this PR.

Reply:
Discussed in today's workgroup meeting, it was decided to go with `messaging.operation.type` and `messaging.operation.name` as proposed above.

Reply:
@lmolkova I changed the PR as discussed, please have a look.